python/lsst/rubintv/production/rubinTv.py

```diff
@@ -58,6 +58,7 @@
                                         getAzimuthElevationDataForEvent,
                                         )
 from lsst.summit.utils.efdUtils import clipDataToEvent, makeEfdClient
+from lsst.summit.utils.auxtel.mount import hasTimebaseErrors
```

Comment:
Minor point, but add this import alphabetically.

Reply:
I mean, the rest are a total mess, but sure 😄 (`isort` and `black` coming in the fairly near term for this stuff too btw)